crates/re_arrow_store/src/store_dump.rs

```diff
@@ -89,7 +89,7 @@ impl DataStore {
                     col_timelines: [(*timeline, col_time.iter().copied().map(Some).collect())]
                         .into(),
                     col_entity_path: std::iter::repeat_with(|| table.ent_path.clone())
-                        .take(table.num_rows() as _)
+                        .take(bucket.num_rows() as _)
```

Comment:
though that's a reentrant read and and we know where that leads with parking_lot so... let's not